SITCOMTN-076.tex

```diff
@@ -49,6 +53,325 @@
+%the students and their academic advisors should consult with the SIT-Com leadership well in advance of circulating their draft to anyone outside the Project.
+% to discuss the expected commissioning time.
+%of their expected graduation.
+
+\section{Peer-reviewed Journal Publications}
+\label{publications}
+
+The Project Publication Policy \citedsp{LPM-162} covers all publications that describe Project-funded work to design, develop, construct, commission, or operate the Observatory, and all publications based on access to non-public intellectual property of the Project or proprietary information related to the Project.
+
+Rubin Observatory is drafting a set of \textbf{Construction Papers} to be published in peer-reviewed journals as part of the body of documentation to describe the technical and scientific performance of the as-built system.
+The Construction Papers describe infrastructure work by Project team members and are thus covered by the Project Publication Policy.
+Construction Papers that describe scientific performance will be completed by the Data Preview 2 release date in order to support the community to write science papers based on Early Science data products.
+
+Members of the Project team might also contribute to science papers that use released commissioning data from the Early Science program.
+The Rubin Data Policy \citedsp{RDO-013} states:
+
+\begin{emph}
+  {DPOL-516 Science Data from Commissioning: Scientific analysis of the commissioning data will be an integral and necessary part of the science verification process.
+  All commissioning data used for science will be released to all LSST Users prior to any publication by anyone.
+  Members of the Project team may not submit science papers to a journal and/or the arXiv based on commissioning data prior to the release of those data to LSST Users, but they may undergo the Rubin Observatory Publication Board process (this board is part of the construction project, not operations) in advance of the release of those data.}
+\end{emph}
+
+%, unless those papers have Project team co-authors and include previously unpublished aspects of Observatory science performance.
+%In that scenario, the specific parts of the paper that
+%also include previously unpublished aspects of Observatory science performance, which must be reviewed by the Project.
+
+%Members of the Project team are free to collaborate with others beyond the Project team on science papers that use Early Science data.
+%Science paper processes, including reviews and approvals, are in the purview of the LSST Science Collaborations.
+
+Science papers that rely only on released Early Science data products are NOT covered by the Project Publication Policy.\footnote{Authors should consult with the Rubin Observatory Publication Manager if they are uncertain whether a manuscript in preparation would be classified as infrastructure work covered by the Project Publication Policy.}
```

Comment:
As I understand, with the exception of the Alert Stream, the Early Science data products are released to the LSST Data Rights Holders, rather than being publicly released. Emphasize "released"